tests/test_database.py

```diff
@@ -339,6 +344,140 @@ def test_insert_many_gyros(database, many_gyros):
 			assert result['y'] == y
 			assert result['z'] == z
 
+def test_insert_many_accels(database, many_accels):
+
+	accel_id = many_accels[1]["accel_id"]
+	description = many_accels[1]["description"]
+	patient_id = many_accels[0]["patient_id"]
```

Comment:
these should all be index 1 for all of the insert many tests. I made a copy and paste error on mine. If you can fix mine too that would be great.

Reply:
Done.